src/rate_limiter/src/lib.rs

```diff
@@ -93,13 +93,18 @@ pub enum BucketReduction {
 pub struct TokenBucket {
     // Bucket defining traits.
     size: u64,
-    // Initial burst size (number of free initial tokens, that can be consumed at no cost)
-    one_time_burst: u64,
+    // Initial burst size.
+    initial_one_time_burst: u64,
```

Comment:
It is not clear to me why this was added and how it differs from the existing `one_time_burst`. Would you mind providing more detail?

Reply:
`one_time_burst` is an internal member that is mutated during operation - we need to store its initial config value (new member `initial_one_time_burst` to be able to generate config from object.

Reply:
Got it, thanks!